packages/components/src/components/money/money.jsx

```diff
@@ -1,9 +1,8 @@
-import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { formatMoney } from '@deriv/shared';
 
-const Money = ({ amount, className, currency = 'USD', has_sign, should_format = true }) => {
+const Money = ({ amount, className, currency = 'USD', has_sign, should_format = true, show_currency = false }) => {
```

Comment:
Can we remove the default value of `currency` and show the currency value only if it is passed? By this way we can get rid of `show_currency` flag.

For `formatMoney` function call, we can pass a default value if it is not available.
`formatMoney(currency || 'USD', value, true)`

Reply:
Ignore the above comment, I assume this for passing in the currency for formatting and not showing it.